src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationQueryObjectType.cs

```diff
@@ -13,6 +18,27 @@ public LocalizationQueryObjectType(IStringLocalizer<LocalizationQueryObjectType>
 
             Field(x => x.Culture).Description(S["The culture for your content item."]);
             Field(x => x.LocalizationSet).Description(S["The localization set for your content item."]);
+
+            Field<ListGraphType<ContentItemType>, IEnumerable<ContentItem>>()
+                .Name("Localizations")
+                .Description(S["The localizations of the content item."])
+                .Argument<StringGraphType, string>("culture", "the culture of the content item", null)
+                .ResolveAsync( async x =>
+                {
+                    var culture = x.GetArgument<string>("culture");
+                    var context = (GraphQLContext)x.UserContext;
+                    var contentLocalizationManager = context.ServiceProvider.GetService<IContentLocalizationManager>();
+
+                    if (culture != null)
+                    {
+                        return new List<ContentItem>
+                        {
+                            await contentLocalizationManager.GetContentItemAsync(x.Source.LocalizationSet, culture)
+                        };
+                    }
+
+                    return await contentLocalizationManager.GetItemsForSetAsync(x.Source.LocalizationSet);
```

Comment:
same as above, can this return null ?

Reply:
It returns an empty list if not results are found (the implementation calls ToListAsync() on an IEnumerable)